src/scenekit.cs

```diff
@@ -121,6 +121,11 @@ interface SCNAnimatable {
 		[Mac (10,10)]
 		[Export ("removeAnimationForKey:fadeOutDuration:")]
 		void RemoveAnimation (NSString key, nfloat duration);
+
+		[TV (10, 0), Mac (10, 12), iOS (10, 0)]
+		[Abstract]
```

Comment:
I think you drew the incorrect conclusions when reading the existing code:

* When we were creating unified we could break compatibility to correct *mistakes* (including missing required on protocol members). That's why `[Abstract]` are only in `XAMCORE_2_0`, i.e. we did not want to break classic;

* However this is not a pattern of how to add *new* `required` members, you can't follow that suit;

* It's also not a mistake (at least not from us). It's a breaking change from Apple that ObjC just does not care as much as we do (our type loader will throw if it see a non-abstract type that does not implement all abstract members);

* Finally we're **only** dropping classic for iOS, not for macOS. As far as bindings are concerned this won't change much how things needs to be done (and `XAMCORE_2_0` remains needed);

> Let me know if you want me change it anyways.

yes :)